subsys/usb/usb_device.c

```diff
@@ -942,8 +942,10 @@ static void forward_status_cb(enum usb_dc_status_code status, const u8_t *param)
 	if (status == USB_DC_DISCONNECTED || status == USB_DC_SUSPEND) {
 		if (usb_dev.configured) {
 			usb_cancel_transfers();
```

Comment:
I agree that endpoints should not be disabled on the USB_DC_SUSPEND events. But I think that also the transfers should not be canceled in this case. The device should be able to continue its operation after the bus is resumed. Currently, it cannot without being configured again. See for instance the cdc_acm class: it starts its RX transfer when the device becomes [configured](https://github.com/zephyrproject-rtos/zephyr/blob/5c96de6194a4bb14cca4d0b7b87234d229762b79/subsys/usb/class/cdc_acm.c#L406) and then, when the previous one has completed.